test/conformance/envpropagation_test.go

```diff
@@ -40,31 +39,10 @@ func TestSecretsViaEnv(t *testing.T) {
 	t.Parallel()
 	clients := setup(t)
 
-	secretName := test.AppendRandomString("secret-")
-
-	//Creating test secret
-	secret, err := clients.KubeClient.Kube.CoreV1().Secrets(test.ServingNamespace).Create(&corev1.Secret{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: secretName,
-		},
-		Type: corev1.SecretTypeOpaque,
-		StringData: map[string]string{
-			testKey: testValue,
-		},
-	})
-	if err != nil {
-		t.Fatal(err)
-	}
-	t.Logf("Successfully created test secret: %v", secret)
-
-	defer func() {
-		if err := cleanupSecret(secretName, clients); err != nil {
-			t.Error(err)
-		}
-	}()
+	secretName := "conformance-test-secret"
```

Comment:
I need advice on how to refer to configmap and secret without hardcoding their names

Reply:
There may be a better place for them, but I would just move them to `test/e2e_flags.go` for now. This puts them in the same place as the namespace strings that will get created by running `kubectl apply -f test/config`.

Reply:
done